lib/dsp.js

```diff
@@ -0,0 +1,2301 @@
+/* 
```

Comment:
Add this dependency to package.json instead, so we get it from npm: https://www.npmjs.com/package/dspjs